src/Connections/GitlabRepo.ts

```diff
@@ -512,6 +563,29 @@ ${data.description}`;
         }
         return false;
     }
+
+    public async provisionerUpdateConfig(userId: string, config: Record<string, unknown>) {
+        const validatedConfig = validateState(config);
+        await this.as.botClient.sendStateEvent(this.roomId, GitLabRepoConnection.CanonicalEventType, this.stateKey, 
+            {
+                ...validatedConfig,
+            }
```

Comment:
`validatedConfig` already is a new object that's not handed to any other function.
It wouldn't need to be redefined here.

Reply:
Whoops, c&p fail.